mobile-app/src/components/Board/Board.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { GameState } from '../../../utils';
+import Board from './Board';
+
+describe('Board', () => {
+  it('Does not throw any error', () => {
+    const gamestate: GameState = {
+      turn: 'white',
+      winner: null,
+      winningPath: null,
+      board: [
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'white' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'black' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'black' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'white' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+      ],
+    }
+    expect(() => renderer.create(<Board gameState={gamestate} />)).not.toThrowError();
+  });
+  it('Render a board with the correct nb of cells', () => {
+    const gamestate: GameState = {
+      turn: 'white',
+      winner: null,
+      winningPath: null,
+      board: [
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'white' }, { value: 'empty' }],
+        [{ value: 'empty' }, { value: 'empty' }, { value: 'empty' }, { value: 'empty' }],
+      ],
+    }
+
+    const board = renderer.create(<Board gameState={gamestate} />).toJSON();
+
+    expect(board.children[0].children.length).toBe(23);

```

Comment:
I think i've just not understood how it works but your gameState.board contain only 12 values, how can you expect 23 here ?
you should maybe add a little comment here to explain how it works.